hidden-posts.php

```diff
@@ -180,6 +181,48 @@ public function custom_column_style() {
 		print( '<style> .fixed .column-visibility { width: 5.5em; } </style>' );
 	}
 
+	/**
+	 * Add custom filter to the admin columns.
+	 *
+	 * @param array $views The original array with view links.
+	 *
+	 * @return array The updated array with view links.
+	 */
+	public function custom_column_filter( $views ) {
+
+		if ( ! is_admin() ) {
+			return $views;
+		}
+
+		if ( isset( $_GET['post_type'] ) && 'post' !== $_GET['post_type'] ) { // phpcs:ignore
+			return $views;
+		}
+
+		if ( ! count( get_option( 'hidden-posts' ) ) ) {
```

Comment:
I think this (and the next instance) need to set a default value, or else if no posts have yet been hidden, it will throw a warning about an uncountable value.

Reply:
Thanks for pointing this out. When testing, I did not see a warning by default, as the call `get_option( 'hidden-posts' )` returns an empty array when no hidden posts exist. However, when deleting the option `hidden-posts` directly within  the database, the following warning appears:

```
( ! ) Warning: count(): Parameter must be an array or an object that implements Countable in /Users/nielslange/Development/plugins/hidden-posts/hidden-posts.php on line 201
```

As suggested by you, I've added a default value for both `get_option()` function calls to prevent a warning from being thrown.